### ISSUE TITLE
Thread sidebar on the right of media in desktop view upon expanding media

### ISSUE THREAD

**Issue by aquariusdev** [nolanlawson/pinafore#1742]
### Pitch

In desktop view, when clicking and expanding media (images/video/audio), a comment bar (containing the attached thread and all fav/boost/reply/bookmark/etc buttons) can display on the right side of media. The "show more" button for images in the containing toot would be hidden in this view, and sensitive media would have to be revealed first before opening in expanded view + comment bar.

Also a button show or hide the comment bar, and a settings option to either hide or show the comment bar by default.

And allow users to reply to a user within the comment bar.

### Motivation

Allowing on desktop for discussion and interaction in the replies while viewing media at the same time.

**Comment by nolanlawson** [comment]
Nice idea! Makes sense for the desktop view

**Comment by aquariusdev** [comment]
> Nice idea! Makes sense for the desktop view

Yep, I have mockups of what it would look like in multi-pane Mastodon, although I doubt they'll look at it before the year is out: https://github.com/tootsuite/mastodon/issues/13530
